Makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: install tidy deps \
+	lint lint-md lint-go \
+	lint-fix lint-md-fix
+
+ifeq ($(OS),Windows_NT)
+wharf.exe:
+	go build -o wharf.exe
+else
+wharf:
+	go build -o wharf
+endif
+
+install:
+	go install
```

Comment:
Conventionally, `make install` means "install this application on your computer", and you usually need to do `sudo make install`. (Go installs into `~/go/bin/...` so it can do without sudo) Docs: https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html#Standard-Targets

I've added `make deps` because I saw someone use that target before. Now I cannot find the source. When searching I did find this https://github.com/netlify/gotrue/blob/87f068d/Makefile#L12-L15 but it's not where I originally found the usage of a `deps` target.